### PR TITLE
check verbose before print executed instructions

### PR DESCRIPTION
**Pull request by butastur-rtos**
check verbose before print executed instructions...
only add if(verbose){...}

### CODE REVIEW

File: amacc.c [hunk 1]
```diff
@@ -141,9 +141,11 @@ void next()
             if (src) {
                 printf("%d: %.*s", line, p - lp, lp);
                 lp = p;
-                while (le < e) {
-                    printf("%8.4s", &ops[*++le * 5]);
-                    if (*le <= ADJ) printf(" %d\n", *++le); else printf("\n");
+                if(verbose){
```
Comment (jserv) [inline amacc.c]: Follow consistent coding style. Should be `if (verbose) {`.

**Comment by jserv** [comment]
Please do read [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) carefully and rework Git commit messages.

**Comment by butastur-rtos** [comment]
After I finished studying document I will rework the Git commit messages

**Comment by jserv** [comment]
@butastur-rtos, use `git rebase -i` to squash commits.

**Pull request merged**